demo/index.html

```diff
@@ -221,32 +227,6 @@
       </demo-snippet>
 
       <h3>
-        Tabs can be used in a <code>paper-toolbar</code>.
```

Comment:
There's a demo for that in iron-selector; do people specifically ask about it in the context of paper-tabs?